src/content/app/tools/blast/components/blast-submission-header/BlastSubmissionHeader.tsx

```diff
@@ -115,9 +116,12 @@ export const BlastSubmissionHeader = (props: Props) => {
     <>
       <div className={styles.grid}>
         <div>{blastProgram}</div>
-        <div className={styles.submissionName}>
-          {/* placeholder for submission name */}
-        </div>
+        {jobName && (
+          <div className={styles.submissionName}>
+            <span className={styles.submissionIdLabel}>Submission name</span>
```

Comment:
This class name is a bit misleading since there is another `submissionIdLabel` below meant for the actual `submissionId`. How about `submissionNameLabel` instead?

Reply:
:heavy_check_mark: